radio/src/gui/480x272/widgets/outputs.cpp

```diff
@@ -36,18 +36,14 @@ class OutputsWidget: public Widget
 
     uint8_t drawChannels(uint16_t x, uint16_t y, uint16_t w, uint16_t h, uint8_t firstChan)
     {
-      char chanString[] = "CH32";
+      char chanString[] = "CH32 123456";
```

Comment:
~~Looks like we had a buffer overflow in `char chanString[]` before.~~ my bad, it was OK.

The new size is actually too big now. This line can be changed back to the original.